lib/models/order.rb

```diff
@@ -16,14 +16,28 @@ class Order < Model
         :amount_orig => 7,
         :type => 8,
         :type_prev => 9,
+        :mts_tif => 10,
+        # placeholder
         :flags => 12,
         :status => 13,
+        # placeholder
+        # placeholder
         :price => 16,
         :price_avg => 17,
         :price_trailing => 18,
         :price_aux_limit => 19,
-        :notify => 23,
-        :placed_id => 25
+        # placeholder
+        # placeholder
+        # placeholder
+        # placeholder
+        :notify => 24,
```

Comment:
This doesn't seem correct, it is at index 23 in the docs & `bfx-api-node-models`, can you confirm?

Reply:
according to the docs:

```
[
  #    ID, 1
  #    GID, 2
  #    CID, 3
  #    SYMBOL, 4
  #    MTS_CREATE, 5 
  #    MTS_UPDATE, 6
  #    AMOUNT, 7
  #    AMOUNT_ORIG, 8 
  #    TYPE, 9
  #    TYPE_PREV, 10
  #    MTS_TIF, 11
  #    _PLACEHOLDER, 12
  #    FLAGS, 13
  #    ORDER_STATUS, 14
  #    _PLACEHOLDER, 15
  #    _PLACEHOLDER, 16
  #    PRICE, 17
  #    PRICE_AVG, 18
  #    PRICE_TRAILING, 19
  #    PRICE_AUX_LIMIT, 20
  #    _PLACEHOLDER, 21
  #    _PLACEHOLDER, 22
  #    _PLACEHOLDER, 23
  #    _PLACEHOLDER, 24
  #    HIDDEN, 25
  #    PLACED_ID, 26
  #    _PLACEHOLDER, 27
  #    _PLACEHOLDER, 28
  #    ROUTING, 29
  #    _PLACEHOLDER, 30
  #    _PLACEHOLDER, 31
  #    META 32
  #  ],
```

Reply:
so by that, hidden (notify) is 24 (I forgot to zero index)

Reply:
Ignore this - turns out the docs is missing the `NOTIFY` field in the v2 rest response which confused met. Fixed now